### DISCUSSION TITLE
Welcome to onenord-jetbrains Discussions!

### DISCUSSION THREAD

**Discussion by rmehri01** [rmehri01/onenord-jetbrains#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by h45h74x** [comment]
Hi there, 

I'm a Nord theme enjoyer and happened to stumble upon One Nord. Great idea combining One Dark and Nord, it really combines the best of both worlds. 
I love the increased contrast and vibrancy of the colors! :)

Are there any plans to release this color scheme for Visual Studio or Visual Studio Code? I already got it for neovim.

**Reply by rmehri01** [reply]
Hey, thanks I'm glad you like the theme! I don't really use VSCode that much but I'll consider it. It might be a while before I get to it though since I have some changes planned for the neovim theme 😄

**Reply by h45h74x** [reply]
No rush! It would be great to have it in VS or VS Code, but only having it in neovim and Jetbrains-IDEs is perfectly fine for me until you have the time to maintain another IDE-theme. :)

**Reply by s1e2b3i4** [reply]
Hey, firstly, I really love your theme. I always found the syntax highlighting of the Nord theme a little too minimalistic and the vibrant colors of your combination is a perfect match for me. 

I also use VSCode, so last weekend a sat down and hacked a Version together, which works for VSCode. To achieve this, I started with the original Nord theme and modified the editor / syntax colors, to match yours. 
There is a lot of room for improvements here, and I will be glad for every contribution. 
In the coming days I will publish it to the store. Currently, you have to clone the repo and build a local Plugin. 

Thanks again for your work and I hope this can be a great addition. 

https://github.com/s1e2b3i4/onenord-vscode

**Reply by rmehri01** [reply]
Hey there, thanks for the kind words, I'm glad you like the theme! 

Wow this already looks quite nice, thanks for putting it together! I'd be happy to look over it this weekend and contribute if you need the help 😄

**Reply by s1e2b3i4** [reply]
Hey, sorry for the late reply. 
I'm glad you like it, even at this early stage 😅

I welcome every contribution if you have the time. I will continue working on the theme and will try to match your original JetBrains Theme as good as possible in VSCode.
